modules/core/src/lib/loader-utils/option-utils.js

```diff
@@ -149,7 +151,7 @@ function validateOptionsObject(options, id, log, defaultOptions, deprecatedOptio
   for (const key in options) {
     // If top level option value is an object it could options for a loader, so ignore
     const isSubOptions = !id && isObject(options[key]);
-    if (!(key in defaultOptions)) {
+    if (!(key in defaultOptions) && !(key === 'baseUri' && !id)) {
```

Comment:
Ugh! FYI, the reason I haven't dealt with this warning on master (we fixed it on 2.3-release you can check what we did there) is that I wanted to refactor this in the `glTFLoader`

I believe that the url is available in the loader context parameter `parse(data, options, context)` so it might be possible to drop the need for this parameter completely.